pkgs/development/python-modules/graphite-web/default.nix

```diff
@@ -28,6 +28,12 @@ buildPythonPackage rec {
     ./update-django-tagging.patch
   ];
 
+  postPatch = ''
+    # https://github.com/graphite-project/graphite-web/pull/2701
+    substituteInPlace setup.py \
+      --replace 'scandir' 'scandir; python_version < "3.5"'
```

Comment:
Hi,

This looks good. I just fear that when your MR lands upstream, this substitute will still match and produce a bad substitution.

Maybe something like this could prevent this:

```sh
substituteInPlace setup.py \
  --replace "'scandir'" "'scandir; python < \"3.5\"'"
```

Reply:
This is a scenario where I would prefer a plain old patch, because the patch will just fail to apply and it will be obvious what goes wrong, but since `substituteInPlace` is usually preferred, I guess that it would be safer to have substitute that does nothing when this is fixed upstream.

Reply:
I also prefer patches, but in doesn't apply cleanly in this case.